src/platform/platform.h

```diff
@@ -130,11 +140,18 @@ void platform_toggle_windowed_mode();
 void platform_set_cursor(uint8 cursor);
 void platform_refresh_video();
 void platform_process_messages();
-int platform_scancode_to_rct_keycode(int sdl_key);
 void platform_start_text_input(utf8 *buffer, int max_length);
 void platform_stop_text_input();
 void platform_get_date(rct2_date *out_date);
 void platform_get_time(rct2_time *out_time);
+bool platform_keypress_equals(keypress k1, keypress k2);
+bool platform_keypress_in_update(keypress k);
+bool platform_check_alt(void);
+bool platform_check_ctrl(void);
+bool platform_check_gui(void);
+bool platform_check_shift(void);
+#define CHECK_PLACE_OBJECT_MOD_COPY_Z platform_check_ctrl()
```

Comment:
Can you just use the actual function call instead of this macro. I don't think the macro really adds anything.